### PR TITLE
chore: replace webpack for esbuild

### PR DESCRIPTION
**Pull request by wesleimp**
### Changes

Below you'll find a checklist. For each item on the list, check one option and delete the other.

### Tests
- [x] This PR does not require tests

### Changelog
- [x] This PR does not make a user-facing change

### Documentation
- [x] This change does not need a documentation update

### Dark mode
- [x] The UI has been tested both in dark and light mode

### CODE REVIEW

File: assets/tailwind.config.js [hunk 1]
```diff
@@ -1,9 +1,10 @@
 const colors = require('tailwindcss/colors')
 
 module.exports = {
-  purge: [
+  content: [
```
Comment (wesleimp) [inline assets/tailwind.config.js:4]: Reference:
https://tailwindcss.com/docs/upgrade-guide#configure-content-sources

File: config/config.exs [hunk 2]
```diff
@@ -39,4 +39,24 @@ config :plausible, :user_agent_cache,
...
+  version: "0.14.41",
+  default: [
+    args:
+      ~w(js/app.js --bundle --target=es2017 --outdir=../priv/static/assets --external:/fonts/* --external:/images/*),
+    cd: Path.expand("../assets", __DIR__),
+    env: %{"NODE_PATH" => Path.expand("../deps", __DIR__)}
+  ]
+
+config :tailwind,
+  version: "3.1.6",
```
Comment (ukutaht) [inline config/config.exs:52]: We currently use tailwind version 2.1.2. Let's keep it the same instead of upgrading versions as well

**Review by ukutaht — COMMENTED** [review]
Thanks @wesleimp ! Sorry for the delay in getting back on this

I don't see any changes to assets/package.json. We should be able to remove webpack and tailwind related packages from there as well, no?

**Comment by ukutaht** [comment]
Closing in favor of https://github.com/plausible/analytics/pull/3320

**Pull request closed without merging**